tsk/img/img_open.cpp

```diff
@@ -270,10 +273,11 @@ img_open_detect_type(
 }
 
 TSK_IMG_INFO* img_open(
-    int num_img,
-    const TSK_TCHAR* const images[],
-    TSK_IMG_TYPE_ENUM type,
-    unsigned int a_ssize
+  int num_img,
+  const TSK_TCHAR* const images[],
+  TSK_IMG_TYPE_ENUM type,
+  unsigned int a_ssize,
+  [[maybe_unused]] const TSK_IMG_OPTIONS* opts
```

Comment:
I think that we should not be changing indents. It looks like some of the code is indented at 4 and others indented at 2. In the future, can we keep things consistent?

Reply:
Yes, it would be great if everything used 2 spaces, and no tabs.

Reply:
I agree with you on no tabs, but I prefer 4 spaces. However, I do not think that we currently have a coding standards document.